### ISSUE TITLE
missing Tooltips content

### ISSUE THREAD

**Issue by Duchneu** [shagu/pfUI#951]
When I open my spell book and jobs pannel there is some missing informations when overing my cursor on icons. And there is missing  

ERROR: Interface\AddOns\pfUI\libs\libcast.lua:364: bad argument #1 to `strlower' (string expected, got nil)

**Comment by shagu** [comment]
Do you play on a server with custom content? If so, which one?
Do you use a modded gameclient? Which language is it?

**Comment by shagu** [comment]
Please try again with the latest version. Should be fixed by this commit: https://github.com/shagu/pfUI/commit/dcb0ac5b078dfecf76e5ee9566d52251a8d72ed7

**Comment by Duchneu** [comment]
I play on nostalgeek a vanilla server, it's FR langage

**Comment by Duchneu** [comment]
my gameclient is not modded

**Comment by Duchneu** [comment]
only with pfui, and pfui extensions

**Comment by Duchneu** [comment]
maybe I have a mod, it's patch-3.MPQ, it improves models. And I have an ENB. But this is only graphics mods

**Comment by Duchneu** [comment]
ERROR: Interface\AddOns\pfUI\modules\thirdparty.lua:738: <unnamed>:SetParent(): Would create a loop parenting to <unnamed>
ERROR: Interface\AddOns\pfUI\modules\prediction.lua:368: attempt to concatenate local `effect' (a nil value)
it happens when I open my spell book, my jobs, my skills pannel

**Comment by Duchneu** [comment]
do you want screenshots?

**Comment by shagu** [comment]
Yes. Please `/rl` and scroll up your chat to show me the first error messages appearing.

**Comment by Duchneu** [comment]
in my chat

**Comment by shagu** [comment]
It's lower-case "L", not "1" ;)

**Comment by Duchneu** [comment]
![image](https://user-images.githubusercontent.com/61423227/75169557-0f49ec80-5729-11ea-8949-2eccae80e52a.png)

**Comment by shagu** [comment]
Do you see any more errors right after logging in? The topmost is always the most interesting one. Also, please disable WIM for the moment.

**Comment by Duchneu** [comment]
there is nothing when I log

**Comment by Duchneu** [comment]
only when I open pannels

**Comment by shagu** [comment]
Please try again with the latest version. Some of the errors should be fixed by https://github.com/shagu/pfUI/commit/4a6aebfae6ba02188cf51223e23fc6dcdd541ec1

Also please disable WIM for the next screenshot (if any required).

**Comment by Duchneu** [comment]
![image](https://user-images.githubusercontent.com/61423227/75171612-2fc77600-572c-11ea-8e40-a8a9e671fafa.png)
there is no more ERROR message but still have invisible menus

**Comment by shagu** [comment]
Try to delete the Gamecache:
1. Close the Game
2. Delete the WDB directory inside you Wow folder.
3. Start the Game again

**Comment by Duchneu** [comment]
nothing :/

**Comment by Duchneu** [comment]
at the start

**Comment by Duchneu** [comment]
no error message, but still have no menu

**Comment by shagu** [comment]
How does it look like when pfUI is disabled?

**Comment by Duchneu** [comment]
wow vanilla

**Comment by Duchneu** [comment]
it's missing when I disable pfui

**Comment by shagu** [comment]
I'm afraid I can't help when it is a client issue. Maybe the server doesn't have proper localization?

**Comment by shagu** [comment]
I made a character on your server and started to level. When I reached Level 5, I went to Orgrimmar in order to pick leatherworking as you do:

![nostalgeek](https://user-images.githubusercontent.com/11586807/75183758-76749a80-5743-11ea-9efa-3db4d42a3c31.png)

I did all that inside the french gameclient and there was no issue at all. So there must be something wrong on your end. I suggest to download a fresh client.
